### PR TITLE
Add page for JuMP-dev III

### PR DESCRIPTION
**Pull request by odow**
Still to do:
- [x] sign-up form
- [x] deadline for talk submission

### CODE REVIEW

File: meetings/santiago2019/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+# The Third Annual JuMP-dev Workshop
+
+March 12-14, 2019, Santiago, Chile.
+
+This workshop is the third JuMP-dev workshop after the [first](/meetings/mit2017/)
+and [second](/meetings/bordeaux2018/) JuMP-dev workshops.
+
+**To register your attendance at the workshop, (and optionally submit a proposal
```
Comment (joehuchette) [inline meetings/santiago2019/index.md]: Maybe comment that registration is free?

File: meetings/santiago2019/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Oscar Dowson (Northwestern)
+- Benoît Legat (UCLouvain)
+- Joaquim Garcia (PSR & PUC-Rio)
+- Juan Pablo Vielma (MIT)
+- Victor Albornoz (Universidad Técnica Federico Santa María; Local Organization)
+- Gustavo Angulo (Pontificia Universidad Católica de Chile; Local Organization)
+- Rodolfo Carvajal (Universidad Adolfo Ibañez; Local Organization)
+- Diego Moran (Universidad Adolfo Ibañez; Local Organization)
+
+Contact oscar.dowson at northwestern.edu or jump-dev-committee at
```
Comment (joehuchette) [inline meetings/santiago2019/index.md]: You can change this to my email if you'd like :)

File: meetings/santiago2019/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+for a talk), [use this form](https://goo.gl/forms/2nEljULHwC3zzcE13).**
+
+## Outline
+
+The purpose of the workshop is to bring together students,
+researchers, and practitioners with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuMP.jl)
+and related packages. The workshop is open for anybody to attend. In particular,
+we invite new contributors and those who have not met the core development team.
+
+The workshop is structured as a three-day event. The first day will begin with a
```
Comment (joehuchette) [inline meetings/santiago2019/index.md]: Update to: 

> The workshop will include hands-on tutorials (ranging from no-experience necessary, to advanced usages of JuMP), talks by participants and core developers, and ample time for collaboration.

**Comment by odow** [comment]
@joehuchette updated. Ready to merge on my end.

**Comment by joehuchette** [comment]
Thanks, @odow!

**Pull request merged**